cadl-extension/src/code-model-builder.ts

```diff
@@ -480,6 +480,9 @@ export class CodeModelBuilder {
 
         if (linkOperation.linkedOperation) {
           // Cadl requires linked operation written before
+          if (!this.operationCache.get(linkOperation.linkedOperation)) {
+            this.processRoute(groupName, linkOperation.linkedOperation);
```

Comment:
I assume this linkedOperation (processed here) will not appear in API seen by user, as it not added to a client?

Reply:
it will not appear to user, I've checked in ADP generated code.